src/app/components/Results/Results.jsx

```diff
@@ -48,17 +48,17 @@ class Results extends React.Component {
 
   getResultsSort() {
     return sortingOpts.map((d, i) => {
-      return (<li role="region" key={i} onClick={e => this.sortResultsBy(e, d.val, d.label)}>
-        {d.label}
+      return (<li role="region" key={i}>
+        <a href="#" onClick={e => this.sortResultsBy(e, d.val, d.label)}>{d.label}</a>
```

Comment:
we need to make sure that normal conventions of website behavior are kept, in spite of us using a javascript framework, in particular back button support